components/provisional-en.html

```diff
@@ -93,9 +93,16 @@ <h3>Provisional feature removal first notice</h3>
 	</tr>
 	<tr>
 		<th>Template <code>.gc-pg-hlpfl</code></th>
-		<td>"Page success widget" design pattern to let users share their experience on the page.</td>
+		<td>
+			<p>"Page success widget" design pattern to let users share their experience on the page.</p>
+			<p>Feature has been revised and renamed to "page feedback form" (<code>.gc-opuft</code>):</p>
+			<ul>
+				<li><a href="../sites/gc-page-feedback/gc-page-feedback-en.html">Page feedback form - Canada.ca site functionality</a></li>
+				<li><a href="../sites/gc-page-feedback/gc-page-feedback-custom-en.html">Page feedback form with custom parameters - Canada.ca site functionality</a></li>
+			</ul>
+		</td>
 		<td>v7.0</td>
-		<td></td>
+		<td>Stable (vX.X.X) (TODO)</td>
```

Comment:
This will need to be filled-in at some point.

It'll represent the first GCWeb release that'll come with the page feedback tool.